### PR TITLE
I completed ui board, stayed only functionality

### PR DESCRIPTION
**Pull request by marat-sahakyan-923**

### CODE REVIEW

File: src/components/table/Table.jsx [hunk 1]
```diff
@@ -1,38 +1,16 @@
...
-} from "../../utils/callLocalStorage";
+import React from "react";
+// import { useDispatch, useSelector } from "react-redux";
+// import {
+//   updatePropertyNames,
+//   updateTableData,
+// } from "../../features/tableDataInfo/tableDataInfoSlice";
+// import setDataIntoStorage, {
+//   getDatainToStorage,
+// } from "../../utils/callLocalStorage";
```
Comment (M7Arman) [inline src/components/table/Table.jsx]: remove or uncomment

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx [hunk 2]
```diff
@@ -26,11 +27,35 @@ function LayoutButtons() {
...
+          name: "New View",
+        };
+      }
       return { id: selectedTabId, type, name: "New View" };
     }
     if (checkName(selectedObject)) {
+      if (type === "board") {
+        return {
+          id: selectedTabId,
+          boardGroup: [{ "": data.find((e) => e.type === "title").data }],
```
Comment (M7Arman) [inline src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx:44]: why key is empty?

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx [hunk 3]
```diff
@@ -26,11 +27,35 @@ function LayoutButtons() {
...
+          type,
+          name: type,
+        };
+      }
       return { id: selectedTabId, type, name: type };
     }
+    if (type === "board") {
+      return {
+        id: selectedTabId,
+        boardGroup: [{ "": data.find((e) => e.type === "title").data }],
```
Comment (M7Arman) [inline src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx:54]: same as above

File: src/components/drawerForCreateItem/newItemDrawer.jsx [hunk 4]
```diff
@@ -36,30 +39,18 @@ export default function CreateNewDrawer() {
...
+        {["Inbox", "Starred", "Send email", "Drafts"].map((text) => (
           <ListItem key={text} disablePadding>
-            <ListItemButton>
+            {/* <ListItemButton>
               <ListItemIcon>
                 {index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
               </ListItemIcon>
               <ListItemText primary={text} />
-            </ListItemButton>
+            </ListItemButton> */}
```
Comment (M7Arman) [inline src/components/drawerForCreateItem/newItemDrawer.jsx]: remove or uncomment

File: src/components/drawerForCreateItem/newItemDrawer.jsx [hunk 5]
```diff
@@ -4,18 +4,21 @@ import Drawer from "@mui/material/Drawer";
...
-import ListItemButton from "@mui/material/ListItemButton";
-import ListItemIcon from "@mui/material/ListItemIcon";
-import ListItemText from "@mui/material/ListItemText";
-import InboxIcon from "@mui/icons-material/MoveToInbox";
-import MailIcon from "@mui/icons-material/Mail";
+// import ListItemButton from "@mui/material/ListItemButton";
+// import ListItemIcon from "@mui/material/ListItemIcon";
+// import ListItemText from "@mui/material/ListItemText";
+// import InboxIcon from "@mui/icons-material/MoveToInbox";
+// import MailIcon from "@mui/icons-material/Mail";
```
Comment (M7Arman) [inline src/components/drawerForCreateItem/newItemDrawer.jsx]: remove or uncomment

File: src/components/board/boardList/BoardList.jsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <div>
+      {Object.values(titles)[0].map((item, index) =>
+        index !== editTitleByIndex ? (
+          <div key={uuidv4()} className={style.board_btn}>
+            <div>{item.value || "Untitled"}</div>
+            <div className={style.board_icon_container}>
+              <EditIcon onClick={() => setEditTitleByIndex(index)} />
+              <div />
```
Comment (M7Arman) [inline src/components/board/boardList/BoardList.jsx:21]: run formatter

**Review by M7Arman — APPROVED** [review]

File: src/components/table/Table.jsx (hunk 1 above)
Comment (marat-sahakyan-923) [reply src/components/table/Table.jsx]: +

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx (hunk 2 above)
Comment (marat-sahakyan-923) [reply src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx:44]: +

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx (hunk 3 above)
Comment (marat-sahakyan-923) [reply src/components/popovers/createEditTabpopovers/drawSelectedButton/layout/LayoutButtons.jsx:54]: +

File: src/components/drawerForCreateItem/newItemDrawer.jsx (hunk 4 above)
Comment (marat-sahakyan-923) [reply src/components/drawerForCreateItem/newItemDrawer.jsx]: +

**Pull request merged**